scheduler/regression.go

```diff
@@ -0,0 +1,21 @@
+package scheduler
+
+var regressionSchedulerSingleton IScheduler
+
+func Regression() IScheduler {
+	if regressionSchedulerSingleton == nil {
+		regressionSchedulerSingleton = &SimpleScheduler{
+			Scheduler: Scheduler{
+				SchedType:      SchedSimple,
+				InstanceNumber: 0,
+			},
+			Settings: SimpleSchedSettings{
+				ExecutionTime:   -1,
+				Iterations:      1,
+				ConcurrentUsers: 1,
+				RampupDelay:     0.5,
+			},
```

Comment:
init SimpleScheduler as 

```golang
&SimpleScheduler{Scheduler{SchedSimple,0,}, SimpleSchedSettings{.etc}
```

And we would get a compile error if we add an entry = not forget to add that here.

Reply:
Changing them all reads pretty bad 🤔
```
			Scheduler{
				SchedSimple,
				TimeBuffer{},
				0,
				session.ReconnectSettings{},
				nil,
				false,
			},
			SimpleSchedSettings{
				-1,
				1,
				0.5,
				1,
				false,
				false,
			},
```